handlers/validator.go

```diff
@@ -609,9 +609,9 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 			}
 
 			// add attestationStats that are not yet in validator_stats
-			lookback := int64(lastFinalizedEpoch - (lastStatsDay+1)*utils.EpochsPerDay())
-			if lookback > 0 {
-				missedAttestations, err := db.BigtableClient.GetValidatorMissedAttestationHistory([]uint64{index}, lastFinalizedEpoch-uint64(lookback), lastFinalizedEpoch)
+			lookback := int64(validatorPageData.Epoch - (lastStatsDay+1)*utils.EpochsPerDay())
+			if lookback > 0 && validatorPageData.Epoch > 0 {
```

Comment:
Why `&& validatorPageData.Epoch > 0`? If `lookback > 0` then `validatorPageData.Epoch > 0` must also be true, right?

Reply:
You are right, fixed.